src/assets/i18n/fr.json

```diff
@@ -34,14 +34,14 @@
     "Nameservers": "Serveurs DNS",
     "No data for this test": "Il n'y a pas de résultat pour ce test",
     "No result for this query": "Il n'y a pas de résultat pour ce test",
-    "Notice! More info on undelegated test": "Plus d'information sur les test ",
+    "Notice! More info on undelegated test": "Plus d'information sur les tests non délégués",
     "Options": "Options",
     "Page Not Found": "Page introuvable",
     "Parent data fetched with success": "Les données parent ont été récupérées avec succès.",
```

Comment:
parent -> parentes OU du parent

Reply:
Is "parentes" plural of "parent"? A domain has only one parent domain.

Reply:
Fixed (I went for @marc-vanderwal proposal)